listenbrainz/webserver/admin/test_admin.py

```diff
@@ -15,7 +15,7 @@ def test_admin_views_when_not_logged_in(self):
         r = self.client.get('/admin', follow_redirects=True)
         self.assert200(r)
         self.assertNotIn('BDFL Zone', r.data.decode('utf-8'))
-        self.assertIn('You are not authorized', r.data.decode('utf-8'))
+        self.assertTemplateUsed('index.html')
```

Comment:
This changes the meaning of the test a bit too much. Is there anything we can check to make sure the user does not have access to the admin views when not logged in?